models/general/entertainment/en/cricket.txt

```diff
@@ -1,4 +1,4 @@
-matches going on|current matches|score *
+matches going on|current matches|get me a list of matches going on| cricket matches
```

Comment:
You can add more questions to it I think. 
Do you know about matches going on?
Do you know cricket matches going on?
Tell me about matches going on?
Tell me about current cricket matches. 

and same variations for all.